### ISSUE TITLE
Path to GRDIR environment variable configuration

### ISSUE THREAD

**Issue by mkitti** [jheinen/GR.jl#491]
We would eventually like to get back to a state where we can configure GR by just setting the GRDIR environment variable. This is not possible on some platforms since GR_jll eagerly loads its libraries.

1. We need to pin GR_jll for the upcoming release, and possibly retroactively 
https://github.com/jheinen/GR.jl/pull/488

2. We need a release of GR.jl containing the libpath fix https://github.com/jheinen/GR.jl/blob/dd3913dc824af287151c22b4ffc29f3df6b0ed54/src/preferences.jl#L61-L65

3. We need to re-release the dont_dlopen patch to GR_jll
https://github.com/JuliaPackaging/Yggdrasil/pull/5784

I think the easiest way to do this is a major version bump to 0.70 for at least GR_jll and possibly GR.jl

**Comment by jheinen** [comment]
I bumped the GR run-time to 0.70.0 (with all of the above mentioned changes). I did not yet tag a GR.jl yet to allow some testing. Please let me know if there are still problems (ping @t-bltg @mkitti).

- This release should fix random crashes in the test environment.
- `GRDIR` should always be set to the GR root directory to load plugins etc. There is no need to change the Windows `PATH`.
- In case of problems, debug information can be logged using environment variables (`GR_DEBUG`, `GKS_DEBUG`).

**Comment by mkitti** [comment]
Currently we are setting `PATH` to include multiple directories for all the artifact dependencies. For example, for GR_jll we need not only the path where libgr.dll lives but also the directory of the QT libraries I believe. Setting GRDIR only allows us to configure a single directory.

**Comment by t-bltg** [comment]
> I bumped the GR run-time to 0.70.0 (with all of the above mentioned changes). I did not yet tag a GR.jl yet to allow some testing. 

Thanks, I've tested `Plots@1.36.0` locally on `linux x86_64` with `GR@0.70.0` dev'ed out and a local build of `GR_jll@0.70.0`, and have no issue to report.

I cannot test other platforms, and will therefore rely on `Plots` ci.

**Comment by jheinen** [comment]
> Currently we are setting `PATH` to include multiple directories for all the artifact dependencies. For example, for GR_jll we need not only the path where libgr.dll lives but also the directory of the QT libraries I believe. Setting GRDIR only allows us to configure a single directory.

Yes, setting `GRDIR` makes it a bit easier to load the GKS plugins and troubleshoot in case of problems.
